### PR TITLE
Feat(Compiler-Base): Add package config in cargo.toml and README.md.

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

issue #115

#### 2. What is the scope of this PR (e.g. component or file name):

Compiler_base

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [x] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

Add package config in cargo.toml and README.md

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: compiler_base/Cargo.toml [hunk 1]
```diff
@@ -2,6 +2,14 @@
 name = "compiler_base"
 version = "0.1.0"
 edition = "2021"
+authors = ["zongzhe1024@163.com"]
+license = "Apache-2.0"
+description = "A tool for compiler"
```
Comment (Peefy) [inline compiler_base/Cargo.toml:7]: A common domain programming language framework?

Comment (zong-zhe) [reply compiler_base/Cargo.toml]: Fixed

**Review by He1pa — APPROVED** [review]
lgtm

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**